tests/test_command.py

```diff
@@ -95,14 +96,17 @@ def _eq_cmd_output(self, buf, expected, env_token=False, currents=()):
             rev = script.get_revision(_id)
             if _id in currents:
                 rev._db_current_indicator = True
-            assert_lines.append(rev.log_entry)
+            entry = rev.log_entry
+            if os.linesep != '\n':
+                entry = entry.replace('\n', os.linesep)
+            assert_lines.append(entry)
 
         if env_token:
             assert_lines.insert(0, "environment included OK")
 
         eq_(
             buf.getvalue().decode("ascii", "replace").strip(),
```

Comment:
so on this side we are taking "raw" content and converting to something agnostic, so let's make it:

    buf.getvalue().decode("ascii", "replace").replace(os.linesep, "\n").strip()    

and that's it, then we don't need a conditonal above or anything

Reply:
I preferred to change the expected value, but in this case I guess it's best change the actual value